freebasics/static/css/template-base.css

```diff
@@ -309,4 +308,9 @@ h6{
 .button-div a, .button-div input[type="submit"]{
   width: 12em;
   margin: 0 auto;
+  display: block;
+}
+.free-text-form input[type="text"], input[type="number"]{
```

Comment:
the second part applies to all `input[type="number"]`. You should prefix it with `.free-text-form input[type="number"]`